source-map.d.ts

```diff
@@ -61,6 +61,7 @@ export interface MappingItem {
     source: string;
     generatedLine: number;
     generatedColumn: number;
+    lastGeneratedColumn?: number;
```

Comment:
I think it should be 
```
 lastGeneratedColumn: number | null;
```
If I understand code correctly the initial value is [null](https://github.com/mozilla/source-map/blob/ddaac7f00a67aefbf184bae16cb90b30ddc84a67/lib/wasm.js#L9) then it can receive number values including `Infinity`

Reply:
This makes me think it is likely optional:

https://github.com/mozilla/source-map/blob/master/lib/wasm.js#L48-L51

Reply:
@ericmorand @samccone which do you think is the better typing:

```js
lastGeneratedColumn?: number;
```

```js
 lastGeneratedColumn: number | null;
```

Reply:
My interpretation is that `lastGeneratedColumn: number | null;` is correct, I agree with @nikolay-borzov

`lastGeneratedColumn?: number;` is equivalent to `lastGeneratedColumn: number | undefined;` but it is initialized as `null`.

Reply:
Thanks. Happy to update and land.